examples/with-nextjs-and-clerk-auth/src/themes/monite.ts

```diff
@@ -0,0 +1,165 @@
+import type { Components } from '@mui/material/styles/components';
+import type {
+  Palette,
+  PaletteOptions,
+} from '@mui/material/styles/createPalette';
+import type { Theme, ThemeOptions } from '@mui/material/styles/createTheme';
+import type { TypographyOptions } from '@mui/material/styles/createTypography';
+import {
+  paletteDark as basePaletteDark,
+  paletteLight as basePaletteLight,
+  defaultMoniteComponents,
+  defaultMoniteTypography,
+} from '@team-monite/sdk-themes';
+
+const paletteLight: PaletteOptions = {
+  ...basePaletteLight,
```

Comment:
By using destructuring, you break the standard theme gluing mechanism.
In the MUI, in order to extend an external theme with your own styles, you need to pass styles as the second argument to `createTheme(defaultTheme, {/* one more theme */})`.

Check out the [API](https://mui.com/material-ui/customization/theming/#api).

Reply:
Yeah, sure, you can use `deepmerge` at this point to merge different themes. But I see no reason to call `createTheme()` as many times as we import themes. You can do this once: `<ThemeProvider theme={createTheme(theme)} />`. Also, you already call `createTheme()` in [`AppThemeProvider`](https://github.com/team-monite/monite-sdk/blob/e4d41e7e112b6dc96ead64e7d4a78a0531dab3db/examples/with-nextjs-and-clerk-auth/src/components/ThemeRegistry/AppThemeProvider.tsx#L110-L120).